base/202-extension-crd.yaml

```diff
@@ -23,15 +23,33 @@ metadata:
     app.kubernetes.io/part-of: tekton-dashboard
 spec:
   group: dashboard.tekton.dev
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+  preserveUnknownFields: false
+  validation:
+    openAPIV3Schema:
+      type: object
+      # One can use x-kubernetes-preserve-unknown-fields: true at the root
+      # of the schema (and inside any properties, additionalProperties)
+      # to get the traditional CRD behaviour that nothing is pruned, despite
+      # setting spec.preserveUnknownProperties: false.
+      #
+      # See https://kubernetes.io/blog/2019/06/20/crd-structural-schema/
+      # See issue: https://github.com/knative/serving/issues/912
+      x-kubernetes-preserve-unknown-fields: true
```

Comment:
this is preparing for the switch to `apiextensions.k8s.io/v1` so we maintain the existing behaviour from 1.15 and earlier?

Reply:
i think so :)